llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -3671,6 +3671,7 @@ class BoUpSLP {
 
     if (UserTreeIdx.UserTE)
       Last->UserTreeIndices.push_back(UserTreeIdx);
+    Last->dump();
```

Comment:
No, it's a bad solution. 1. It first be emitted only in debug builds and only if -debug is provided. 2. Better to move it to each LLVM_DEBUG, where the messages for entries are printed